pkg/util/trivy/cache.go

```diff
@@ -100,28 +101,28 @@ type Cache interface {
 	Len() int
 }
 
-// TrivyCache holds a generic Cache and implements cache.Cache from Trivy.
-type TrivyCache struct {
+// TCache holds a generic Cache and implements cache.Cache from Trivy.
```

Comment:
I think `TCache` here isn't meaningful. Can we change that ?

Reply:
I agree.. The issue here was that the linter didn't want to have a trivy.TrivyCache, asking rename it to trivy.Cache .. but there is already a such type name .. So, I land with this poor one but, happy to hear if you have suggestions :)

Reply:
Can we rename it as ScannerCache ?

Reply:
Sure, let me change that